core/pom.xml

```diff
@@ -32,6 +32,10 @@ under the License.
 			<artifactId>slf4j-api</artifactId>
 		</dependency>
 		<dependency>
+			<groupId>com.google.guava</groupId>
+			<artifactId>guava</artifactId>
```

Comment:
We should only add the dependency where necesary. You are using guava in the JDBC module, so you should add it as a dependency there instead of the `core` module.